src/main/kotlin/blackjack/domain/card/DefaultCardFactory.kt

```diff
@@ -0,0 +1,17 @@
+package blackjack.domain.card
+
+import blackjack.domain.Card
+import blackjack.domain.CardShape
+import blackjack.domain.CardType
+
+class DefaultCardFactory : CardFactory {
+    override fun createCards(): List<Card> {
+        return createAllShapeCards().shuffled().toMutableList()
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        return createAllShapeCards().shuffled()
```